sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/_client.py

```diff
@@ -8,7 +8,7 @@
 from ._shared import KeyVaultClientBase
 from ._shared.exceptions import error_map as _error_map
 from ._shared._polling import DeleteRecoverPollingMethod, KeyVaultOperationPoller
-from ._models import DeletedKey, KeyVaultKey, KeyProperties
+from ._models import DeletedKey, KeyVaultKey, KeyProperties, KeyReleaseResult
```

Comment:
Maybe this gets renamed later, but this should be `ReleaseKeyResult` to match our naming convention for the method, `release_key`. Same with `ReleaseKeyOptions`, though I suspect you don't have that in favor of `kwargs`.